Reduce use of Vector::uncheckedAppend() throughout the codebase

cdumez opened the pull request:
#### e7149c4d7559e4b663d3f375be37dc5dba82c8ca
<pre>
Reduce use of Vector::uncheckedAppend() throughout the codebase
<a href="https://bugs.webkit.org/show_bug.cgi?id=263002">https://bugs.webkit.org/show_bug.cgi?id=263002</a>

Reviewed by Brent Fulgham.

Reduce use of Vector::uncheckedAppend() throughout the codebase and replace
with more efficient idioms now that uncheckedAppend() has become an alias to
append().

* Source/WTF/wtf/JSONValues.h:
(WTF::containerSize):
* Source/WTF/wtf/URLParser.cpp:
(WTF::URLParser::syntaxViolation):
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUDeviceImpl.cpp:
(WebCore::WebGPU::convertToBacking):
* Source/WebCore/Modules/mediastream/gstreamer/GStreamerPeerConnectionBackend.cpp:
(WebCore::GStreamerPeerConnectionBackend::dispatchPendingTrackEvents):
* Source/WebCore/contentextensions/ContentExtensionActions.cpp:
(WebCore::ContentExtensions::RedirectAction::URLTransformAction::serialize const):
* Source/WebCore/contentextensions/ContentExtensionsBackend.cpp:
(WebCore::ContentExtensions::ContentExtensionsBackend::actionsForResourceLoad const):
* Source/WebCore/contentextensions/DFABytecodeCompiler.cpp:
(WebCore::ContentExtensions::DFABytecodeCompiler::extractJumpTable):
* Source/WebCore/contentextensions/DFAMinimizer.cpp:
* Source/WebCore/contentextensions/MutableRangeList.h:
* Source/WebCore/css/typedom/ComputedStylePropertyMapReadOnly.cpp:
(WebCore::ComputedStylePropertyMapReadOnly::entries const):
* Source/WebCore/cssjit/SelectorCompiler.cpp:
(WebCore::SelectorCompiler::SelectorCodeGenerator::generateElementIsNthLastChildOf):
* Source/WebCore/page/mac/ImageOverlayControllerMac.mm:
(WebCore::ImageOverlayController::updateDataDetectorHighlights):
* Source/WebCore/platform/audio/MultiChannelResampler.cpp:
(WebCore::MultiChannelResampler::MultiChannelResampler):
* Source/WebCore/platform/graphics/ComplexTextController.h:
* Source/WebCore/platform/graphics/cocoa/FontDatabase.cpp:
(WebCore::FontDatabase::collectionForFamily):
* Source/WebCore/platform/graphics/iso/ISOBox.cpp:
(WebCore::ISOBox::parse):
* Source/WebCore/platform/graphics/mac/ComplexTextControllerCoreText.mm:
(WebCore::ComplexTextController::ComplexTextRun::ComplexTextRun):
* Source/WebCore/platform/ios/PlatformEventFactoryIOS.mm:
(WebCore::PlatformTouchEventBuilder::PlatformTouchEventBuilder):
* Source/WebCore/platform/ios/WebCoreMotionManager.mm:
(-[WebCoreMotionManager sendAccelerometerData:]):
(-[WebCoreMotionManager sendMotionData:withHeading:]):
* Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProvider.cpp:
(WebCore::toRTCRtpCapabilities):
* Source/WebCore/rendering/RenderThemeIOS.mm:
(WebCore::RenderThemeIOS::paintCheckboxDecorations):
(WebCore::RenderThemeIOS::adjustedPaintRect const): Deleted.
(WebCore::RenderThemeIOS::baselinePosition const): Deleted.
(WebCore::RenderThemeIOS::isControlStyled const): Deleted.
(WebCore::RenderThemeIOS::adjustMinimumIntrinsicSizeForAppearance const): Deleted.
(WebCore::RenderThemeIOS::adjustRadioStyle const): Deleted.
(WebCore::RenderThemeIOS::paintRadioDecorations): Deleted.
(WebCore::RenderThemeIOS::adjustTextFieldStyle const): Deleted.
(WebCore::RenderThemeIOS::paintTextFieldInnerShadow): Deleted.
(WebCore::RenderThemeIOS::paintTextFieldDecorations): Deleted.
(WebCore::RenderThemeIOS::adjustTextAreaStyle const): Deleted.
(WebCore::RenderThemeIOS::paintTextAreaDecorations): Deleted.
(WebCore::RenderThemeIOS::popupInternalPaddingBox const): Deleted.
(WebCore::canAdjustBorderRadiusForAppearance): Deleted.
(WebCore::RenderThemeIOS::adjustRoundBorderRadius): Deleted.
(WebCore::applyCommonButtonPaddingToStyle): Deleted.
(WebCore::adjustSelectListButtonStyle): Deleted.
(WebCore::RenderThemeMeasureTextClient::RenderThemeMeasureTextClient): Deleted.
(WebCore::adjustInputElementButtonStyle): Deleted.
(WebCore::RenderThemeIOS::adjustMenuListButtonStyle const): Deleted.
(WebCore::RenderThemeIOS::paintMenuListButtonDecorations): Deleted.
(WebCore::RenderThemeIOS::adjustSliderTrackStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSliderTrack): Deleted.
(WebCore::RenderThemeIOS::adjustSliderThumbSize const): Deleted.
(WebCore::RenderThemeIOS::paintSliderThumbDecorations): Deleted.
(WebCore::RenderThemeIOS::paintProgressBar): Deleted.
(WebCore::RenderThemeIOS::sliderTickSize const): Deleted.
(WebCore::RenderThemeIOS::sliderTickOffsetFromTrackCenter const): Deleted.
(WebCore::RenderThemeIOS::adjustSearchFieldStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSearchFieldDecorations): Deleted.
(WebCore::RenderThemeIOS::isSubmitStyleButton const): Deleted.
(WebCore::RenderThemeIOS::adjustButtonLikeControlStyle const): Deleted.
(WebCore::RenderThemeIOS::adjustButtonStyle const): Deleted.
(WebCore::RenderThemeIOS::paintButtonDecorations): Deleted.
(WebCore::shouldUseConvexGradient): Deleted.
(WebCore::RenderThemeIOS::paintPushButtonDecorations): Deleted.
(WebCore::RenderThemeIOS::platformActiveSelectionBackgroundColor const): Deleted.
(WebCore::RenderThemeIOS::platformInactiveSelectionBackgroundColor const): Deleted.
(WebCore::cachedFocusRingColor): Deleted.
(WebCore::cachedInsertionPointColor): Deleted.
(WebCore::RenderThemeIOS::systemFocusRingColor): Deleted.
(WebCore::RenderThemeIOS::platformFocusRingColor const): Deleted.
(WebCore::RenderThemeIOS::insertionPointColor): Deleted.
(WebCore::RenderThemeIOS::autocorrectionReplacementMarkerColor const): Deleted.
(WebCore::RenderThemeIOS::platformAnnotationHighlightColor const): Deleted.
(WebCore::RenderThemeIOS::shouldHaveSpinButton const): Deleted.
(WebCore::RenderThemeIOS::supportsFocusRing const): Deleted.
(WebCore::RenderThemeIOS::supportsBoxShadow const): Deleted.
(WebCore::cssValueSystemColorInformationList): Deleted.
(WebCore::systemColorFromCSSValueSystemColorInformation): Deleted.
(WebCore::systemColorFromCSSValueID): Deleted.
(WebCore::globalCSSValueToSystemColorMap): Deleted.
(WebCore::RenderThemeIOS::cssValueToSystemColorMap): Deleted.
(WebCore::RenderThemeIOS::setCSSValueToSystemColorMap): Deleted.
(WebCore::RenderThemeIOS::setFocusRingColor): Deleted.
(WebCore::RenderThemeIOS::setInsertionPointColor): Deleted.
(WebCore::RenderThemeIOS::systemColor const): Deleted.
(WebCore::RenderThemeIOS::pictureFrameColor): Deleted.
(WebCore::RenderThemeIOS::controlTintColor const): Deleted.
(WebCore::RenderThemeIOS::iconForAttachment): Deleted.
(WebCore::RenderThemeIOS::attachmentIntrinsicSize const): Deleted.
(WebCore::paintAttachmentIcon): Deleted.
(WebCore::paintAttachmentProgress): Deleted.
(WebCore::attachmentBorderPath): Deleted.
(WebCore::paintAttachmentBorder): Deleted.
(WebCore::RenderThemeIOS::paintAttachment): Deleted.
(WebCore::RenderThemeIOS::attachmentStyleSheet const): Deleted.
(WebCore::RenderThemeIOS::extraDefaultStyleSheet): Deleted.
(WebCore::RenderThemeIOS::paintSystemPreviewBadge): Deleted.
(WebCore::RenderThemeIOS::checkboxRadioBorderColor): Deleted.
(WebCore::RenderThemeIOS::checkboxRadioBackgroundColor): Deleted.
(WebCore::RenderThemeIOS::checkboxRadioBackgroundGradient): Deleted.
(WebCore::RenderThemeIOS::checkboxRadioIndicatorColor): Deleted.
(WebCore::RenderThemeIOS::paintCheckboxRadioInnerShadow): Deleted.
(WebCore::RenderThemeIOS::paintCheckbox): Deleted.
(WebCore::RenderThemeIOS::paintRadio): Deleted.
(WebCore::RenderThemeIOS::animationRepeatIntervalForProgressBar const): Deleted.
(WebCore::RenderThemeIOS::paintProgressBarWithFormControlRefresh): Deleted.
(WebCore::RenderThemeIOS::supportsMeter const): Deleted.
(WebCore::RenderThemeIOS::paintMeter): Deleted.
(WebCore::RenderThemeIOS::paintListButton): Deleted.
(WebCore::RenderThemeIOS::paintSliderTicks): Deleted.
(WebCore::RenderThemeIOS::paintSliderTrackWithFormControlRefresh): Deleted.
(WebCore::RenderThemeIOS::colorInputStyleSheet const): Deleted.
(WebCore::RenderThemeIOS::adjustColorWellStyle const): Deleted.
(WebCore::RenderThemeIOS::paintColorWell): Deleted.
(WebCore::RenderThemeIOS::paintColorWellDecorations): Deleted.
(WebCore::RenderThemeIOS::paintMenuListButtonDecorationsWithFormControlRefresh): Deleted.
(WebCore::RenderThemeIOS::adjustSearchFieldDecorationPartStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSearchFieldDecorationPart): Deleted.
(WebCore::RenderThemeIOS::adjustSearchFieldResultsDecorationPartStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSearchFieldResultsDecorationPart): Deleted.
(WebCore::RenderThemeIOS::adjustSearchFieldResultsButtonStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSearchFieldResultsButton): Deleted.
* Source/WebKit/Shared/WebEventConversion.cpp:
(WebKit::WebKit2PlatformTouchEvent::WebKit2PlatformTouchEvent):
* Source/WebKit/Shared/glib/ArgumentCodersGLib.cpp:
(IPC::ArgumentCoder&lt;GRefPtr&lt;GUnixFDList&gt;&gt;::encode):
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm:
(-[WKWebsiteDataStore _fetchDataRecordsOfTypes:withOptions:completionHandler:]):
* Source/WebKit/UIProcess/API/glib/WebKitContextMenuClient.cpp:
* Source/WebKit/UIProcess/Automation/SimulatedInputDispatcher.cpp:
(WebKit::SimulatedInputKeyFrame::keyFrameFromStateOfInputSources):
(WebKit::SimulatedInputKeyFrame::keyFrameToResetInputSources):
* Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp:
(WebKit::WebAutomationSession::evaluateJavaScriptFunction):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm:
(WebKit::LocalAuthenticator::getAssertion):
* Source/WebKit/UIProcess/WebBackForwardList.cpp:
(WebKit::WebBackForwardList::backListAsAPIArrayWithLimit const):
(WebKit::WebBackForwardList::forwardListAsAPIArrayWithLimit const):
* Source/WebKit/UIProcess/gtk/ClipboardGtk4.cpp:
(WebKit::Clipboard::formats):
* Source/WebKit/WebProcess/Automation/WebAutomationSessionProxy.cpp:
(WebKit::WebAutomationSessionProxy::setFilesForInputFileUpload):
* Source/WebKit/WebProcess/WebPage/IPCTestingAPI.cpp:
(WebKit::IPCTestingAPI::JSIPC::serializedEnumInfo):
* Source/WebKit/WebProcess/glib/UserMediaCaptureManager.cpp:
(WebKit::UserMediaCaptureManager::getMediaStreamDevices):

Canonical link: <a href="https://commits.webkit.org/269224@main">https://commits.webkit.org/269224@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/dfae48bf63d586fffa84f020ef0fc7ec0c23e2f5

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/21933 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/22153 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/23000 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/23815 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/20311 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/22178 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/26400 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/22465 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/21394 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/22163 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/21799 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/19026 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/24667 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/18927 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/19880 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/26150 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/19069 "Built successfully and passed tests") | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/19955 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/20102 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/24014 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/21308 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/20533 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/17492 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/25351 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/19883 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/5982 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/5229 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/24090 "Built successfully") | | [✅ 🛠 jsc-mips](https://ews-build.webkit.org/#/builders/24/builds/26624 "Built successfully") | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/20483 "Built successfully") | | [✅ 🧪 jsc-mips-tests](https://ews-build.webkit.org/#/builders/3/builds/5814 "Passed tests") | 
<!--EWS-Status-Bubble-End-->

brentfulgham commented on Source/WebCore/cssjit/SelectorCompiler.cpp:
```diff
@@ -4328,18 +4328,16 @@ void SelectorCodeGenerator::generateElementIsNthLastChildOf(Assembler::JumpList&
-        validSubsetFilters.uncheckedAppend(&nthLastChildOfSelectorInfo);
-    }
+            return std::nullopt;
+        return &nthLastChildOfSelectorInfo;
```
I love how much clearer the `compactMap` version is.

brentfulgham commented on Source/WebCore/platform/graphics/ComplexTextController.h:
```diff
@@ -124,11 +124,14 @@ class ComplexTextController {
         Vector<unsigned, 64> m_glyphEndOffsets;
-        Vector<unsigned, 64> m_coreTextIndices;
+        using CoreTextIndicesVector = Vector<unsigned, 64>;
+        CoreTextIndicesVector m_coreTextIndices;
```
Nit: I think it's more legible to have the `using` declarations together in a separate block.

cdumez replied:
Ok.

brentfulgham commented on Source/WebKit/WebProcess/glib/UserMediaCaptureManager.cpp:
```diff
@@ -67,23 +67,20 @@ void UserMediaCaptureManager::validateUserMediaRequestConstraints(WebCore::Media
-
-        devicesWithCapabilities.reserveInitialCapacity(devices.size());
-        for (auto& device : devices) {
+        auto devicesWithCapabilities = WTF::compactMap(devices, [&](auto& device) -> std::optional<CaptureDeviceWithCapabilities> {
```
Is this returning a `Vector< CaptureDeviceWithCapabilities>`? Or is it now a `Vector<std::optional< CaptureDeviceWithCapabilities>>`?

cdumez replied:
It is returning a `Vector<CaptureDeviceWithCapabilities>` since I am using WTF::compactMap(), compactMap() extracts the optional value and discards the nullopt ones.

brentfulgham reviewed (approved):
I think it looks good, except for the UserMediaCaptureManager.cpp (which only builds on glib, so maybe we don't know if we broke something). r=me

cdumez committed:
Reduce use of Vector::uncheckedAppend() throughout the codebase
https://bugs.webkit.org/show_bug.cgi?id=263002

Reviewed by Brent Fulgham.

Reduce use of Vector::uncheckedAppend() throughout the codebase and replace
with more efficient idioms now that uncheckedAppend() has become an alias to
append().

* Source/WTF/wtf/JSONValues.h:
(WTF::containerSize):
* Source/WTF/wtf/URLParser.cpp:
(WTF::URLParser::syntaxViolation):
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUDeviceImpl.cpp:
(WebCore::WebGPU::convertToBacking):
* Source/WebCore/Modules/mediastream/gstreamer/GStreamerPeerConnectionBackend.cpp:
(WebCore::GStreamerPeerConnectionBackend::dispatchPendingTrackEvents):
* Source/WebCore/contentextensions/ContentExtensionActions.cpp:
(WebCore::ContentExtensions::RedirectAction::URLTransformAction::serialize const):
* Source/WebCore/contentextensions/ContentExtensionsBackend.cpp:
(WebCore::ContentExtensions::ContentExtensionsBackend::actionsForResourceLoad const):
* Source/WebCore/contentextensions/DFABytecodeCompiler.cpp:
(WebCore::ContentExtensions::DFABytecodeCompiler::extractJumpTable):
* Source/WebCore/contentextensions/DFAMinimizer.cpp:
* Source/WebCore/contentextensions/MutableRangeList.h:
* Source/WebCore/css/typedom/ComputedStylePropertyMapReadOnly.cpp:
(WebCore::ComputedStylePropertyMapReadOnly::entries const):
* Source/WebCore/cssjit/SelectorCompiler.cpp:
(WebCore::SelectorCompiler::SelectorCodeGenerator::generateElementIsNthLastChildOf):
* Source/WebCore/page/mac/ImageOverlayControllerMac.mm:
(WebCore::ImageOverlayController::updateDataDetectorHighlights):
* Source/WebCore/platform/audio/MultiChannelResampler.cpp:
(WebCore::MultiChannelResampler::MultiChannelResampler):
* Source/WebCore/platform/graphics/ComplexTextController.h:
* Source/WebCore/platform/graphics/cocoa/FontDatabase.cpp:
(WebCore::FontDatabase::collectionForFamily):
* Source/WebCore/platform/graphics/iso/ISOBox.cpp:
(WebCore::ISOBox::parse):
* Source/WebCore/platform/graphics/mac/ComplexTextControllerCoreText.mm:
(WebCore::ComplexTextController::ComplexTextRun::ComplexTextRun):
* Source/WebCore/platform/ios/PlatformEventFactoryIOS.mm:
(WebCore::PlatformTouchEventBuilder::PlatformTouchEventBuilder):
* Source/WebCore/platform/ios/WebCoreMotionManager.mm:
(-[WebCoreMotionManager sendAccelerometerData:]):
(-[WebCoreMotionManager sendMotionData:withHeading:]):
* Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProvider.cpp:
(WebCore::toRTCRtpCapabilities):
* Source/WebCore/rendering/RenderThemeIOS.mm:
(WebCore::RenderThemeIOS::paintCheckboxDecorations):
(WebCore::RenderThemeIOS::adjustedPaintRect const): Deleted.
(WebCore::RenderThemeIOS::baselinePosition const): Deleted.
(WebCore::RenderThemeIOS::isControlStyled const): Deleted.
(WebCore::RenderThemeIOS::adjustMinimumIntrinsicSizeForAppearance const): Deleted.
(WebCore::RenderThemeIOS::adjustRadioStyle const): Deleted.
(WebCore::RenderThemeIOS::paintRadioDecorations): Deleted.
(WebCore::RenderThemeIOS::adjustTextFieldStyle const): Deleted.
(WebCore::RenderThemeIOS::paintTextFieldInnerShadow): Deleted.
(WebCore::RenderThemeIOS::paintTextFieldDecorations): Deleted.
(WebCore::RenderThemeIOS::adjustTextAreaStyle const): Deleted.
(WebCore::RenderThemeIOS::paintTextAreaDecorations): Deleted.
(WebCore::RenderThemeIOS::popupInternalPaddingBox const): Deleted.
(WebCore::canAdjustBorderRadiusForAppearance): Deleted.
(WebCore::RenderThemeIOS::adjustRoundBorderRadius): Deleted.
(WebCore::applyCommonButtonPaddingToStyle): Deleted.
(WebCore::adjustSelectListButtonStyle): Deleted.
(WebCore::RenderThemeMeasureTextClient::RenderThemeMeasureTextClient): Deleted.
(WebCore::adjustInputElementButtonStyle): Deleted.
(WebCore::RenderThemeIOS::adjustMenuListButtonStyle const): Deleted.
(WebCore::RenderThemeIOS::paintMenuListButtonDecorations): Deleted.
(WebCore::RenderThemeIOS::adjustSliderTrackStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSliderTrack): Deleted.
(WebCore::RenderThemeIOS::adjustSliderThumbSize const): Deleted.
(WebCore::RenderThemeIOS::paintSliderThumbDecorations): Deleted.
(WebCore::RenderThemeIOS::paintProgressBar): Deleted.
(WebCore::RenderThemeIOS::sliderTickSize const): Deleted.
(WebCore::RenderThemeIOS::sliderTickOffsetFromTrackCenter const): Deleted.
(WebCore::RenderThemeIOS::adjustSearchFieldStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSearchFieldDecorations): Deleted.
(WebCore::RenderThemeIOS::isSubmitStyleButton const): Deleted.
(WebCore::RenderThemeIOS::adjustButtonLikeControlStyle const): Deleted.
(WebCore::RenderThemeIOS::adjustButtonStyle const): Deleted.
(WebCore::RenderThemeIOS::paintButtonDecorations): Deleted.
(WebCore::shouldUseConvexGradient): Deleted.
(WebCore::RenderThemeIOS::paintPushButtonDecorations): Deleted.
(WebCore::RenderThemeIOS::platformActiveSelectionBackgroundColor const): Deleted.
(WebCore::RenderThemeIOS::platformInactiveSelectionBackgroundColor const): Deleted.
(WebCore::cachedFocusRingColor): Deleted.
(WebCore::cachedInsertionPointColor): Deleted.
(WebCore::RenderThemeIOS::systemFocusRingColor): Deleted.
(WebCore::RenderThemeIOS::platformFocusRingColor const): Deleted.
(WebCore::RenderThemeIOS::insertionPointColor): Deleted.
(WebCore::RenderThemeIOS::autocorrectionReplacementMarkerColor const): Deleted.
(WebCore::RenderThemeIOS::platformAnnotationHighlightColor const): Deleted.
(WebCore::RenderThemeIOS::shouldHaveSpinButton const): Deleted.
(WebCore::RenderThemeIOS::supportsFocusRing const): Deleted.
(WebCore::RenderThemeIOS::supportsBoxShadow const): Deleted.
(WebCore::cssValueSystemColorInformationList): Deleted.
(WebCore::systemColorFromCSSValueSystemColorInformation): Deleted.
(WebCore::systemColorFromCSSValueID): Deleted.
(WebCore::globalCSSValueToSystemColorMap): Deleted.
(WebCore::RenderThemeIOS::cssValueToSystemColorMap): Deleted.
(WebCore::RenderThemeIOS::setCSSValueToSystemColorMap): Deleted.
(WebCore::RenderThemeIOS::setFocusRingColor): Deleted.
(WebCore::RenderThemeIOS::setInsertionPointColor): Deleted.
(WebCore::RenderThemeIOS::systemColor const): Deleted.
(WebCore::RenderThemeIOS::pictureFrameColor): Deleted.
(WebCore::RenderThemeIOS::controlTintColor const): Deleted.
(WebCore::RenderThemeIOS::iconForAttachment): Deleted.
(WebCore::RenderThemeIOS::attachmentIntrinsicSize const): Deleted.
(WebCore::paintAttachmentIcon): Deleted.
(WebCore::paintAttachmentProgress): Deleted.
(WebCore::attachmentBorderPath): Deleted.
(WebCore::paintAttachmentBorder): Deleted.
(WebCore::RenderThemeIOS::paintAttachment): Deleted.
(WebCore::RenderThemeIOS::attachmentStyleSheet const): Deleted.
(WebCore::RenderThemeIOS::extraDefaultStyleSheet): Deleted.
(WebCore::RenderThemeIOS::paintSystemPreviewBadge): Deleted.
(WebCore::RenderThemeIOS::checkboxRadioBorderColor): Deleted.
(WebCore::RenderThemeIOS::checkboxRadioBackgroundColor): Deleted.
(WebCore::RenderThemeIOS::checkboxRadioBackgroundGradient): Deleted.
(WebCore::RenderThemeIOS::checkboxRadioIndicatorColor): Deleted.
(WebCore::RenderThemeIOS::paintCheckboxRadioInnerShadow): Deleted.
(WebCore::RenderThemeIOS::paintCheckbox): Deleted.
(WebCore::RenderThemeIOS::paintRadio): Deleted.
(WebCore::RenderThemeIOS::animationRepeatIntervalForProgressBar const): Deleted.
(WebCore::RenderThemeIOS::paintProgressBarWithFormControlRefresh): Deleted.
(WebCore::RenderThemeIOS::supportsMeter const): Deleted.
(WebCore::RenderThemeIOS::paintMeter): Deleted.
(WebCore::RenderThemeIOS::paintListButton): Deleted.
(WebCore::RenderThemeIOS::paintSliderTicks): Deleted.
(WebCore::RenderThemeIOS::paintSliderTrackWithFormControlRefresh): Deleted.
(WebCore::RenderThemeIOS::colorInputStyleSheet const): Deleted.
(WebCore::RenderThemeIOS::adjustColorWellStyle const): Deleted.
(WebCore::RenderThemeIOS::paintColorWell): Deleted.
(WebCore::RenderThemeIOS::paintColorWellDecorations): Deleted.
(WebCore::RenderThemeIOS::paintMenuListButtonDecorationsWithFormControlRefresh): Deleted.
(WebCore::RenderThemeIOS::adjustSearchFieldDecorationPartStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSearchFieldDecorationPart): Deleted.
(WebCore::RenderThemeIOS::adjustSearchFieldResultsDecorationPartStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSearchFieldResultsDecorationPart): Deleted.
(WebCore::RenderThemeIOS::adjustSearchFieldResultsButtonStyle const): Deleted.
(WebCore::RenderThemeIOS::paintSearchFieldResultsButton): Deleted.
* Source/WebKit/Shared/WebEventConversion.cpp:
(WebKit::WebKit2PlatformTouchEvent::WebKit2PlatformTouchEvent):
* Source/WebKit/Shared/glib/ArgumentCodersGLib.cpp:
(IPC::ArgumentCoder<GRefPtr<GUnixFDList>>::encode):
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm:
(-[WKWebsiteDataStore _fetchDataRecordsOfTypes:withOptions:completionHandler:]):
* Source/WebKit/UIProcess/API/glib/WebKitContextMenuClient.cpp:
* Source/WebKit/UIProcess/Automation/SimulatedInputDispatcher.cpp:
(WebKit::SimulatedInputKeyFrame::keyFrameFromStateOfInputSources):
(WebKit::SimulatedInputKeyFrame::keyFrameToResetInputSources):
* Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp:
(WebKit::WebAutomationSession::evaluateJavaScriptFunction):
* Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm:
(WebKit::LocalAuthenticator::getAssertion):
* Source/WebKit/UIProcess/WebBackForwardList.cpp:
(WebKit::WebBackForwardList::backListAsAPIArrayWithLimit const):
(WebKit::WebBackForwardList::forwardListAsAPIArrayWithLimit const):
* Source/WebKit/UIProcess/gtk/ClipboardGtk4.cpp:
(WebKit::Clipboard::formats):
* Source/WebKit/WebProcess/Automation/WebAutomationSessionProxy.cpp:
(WebKit::WebAutomationSessionProxy::setFilesForInputFileUpload):
* Source/WebKit/WebProcess/WebPage/IPCTestingAPI.cpp:
(WebKit::IPCTestingAPI::JSIPC::serializedEnumInfo):
* Source/WebKit/WebProcess/glib/UserMediaCaptureManager.cpp:
(WebKit::UserMediaCaptureManager::getMediaStreamDevices):

Canonical link: https://commits.webkit.org/269224@main
